### ISSUE TITLE
MEXC - fetch_deposit_address returns "dictionary update sequence element #0 has length 1; 2 is required"

### ISSUE THREAD

**Issue by moqiguzhu** [ccxt/ccxt#10576]
- OS: centos
- Programming Language version: Python
- CCXT version: 1.61.16
- Python version: 3.6

```
exchange = ccxt.mexc({
    'enableRateLimit': True,
    'apiKey': 'xxxx',
    'secret': 'xxx',
    'password': 'xxxx',
})
params  = {}
exchange.fetch_deposit_address(code="USDT", params = {})
```

```
---> 10 exchange.fetch_deposit_address(code="USDT", params = {})
     11 
     12 # exchange2ccxtexchange["kucoin"].checkRequiredCredentials()

~/anaconda3/envs/py36/lib/python3.6/site-packages/ccxt/mexc.py in fetch_deposit_address(self, code, params)
   1205         rawNetwork = self.safe_string(params, 'network')
   1206         params = self.omit('network')
-> 1207         response = self.fetch_deposit_addresses_by_network(code, params)
   1208         networks = self.safe_value(self.options, 'networks', {})
   1209         network = self.safe_string(networks, rawNetwork, rawNetwork)

~/anaconda3/envs/py36/lib/python3.6/site-packages/ccxt/mexc.py in fetch_deposit_addresses_by_network(self, code, params)
   1178             'currency': currency['id'],
   1179         }
-> 1180         response = self.spotPrivateGetAssetDepositAddressList(self.extend(request, params))
   1181         #
   1182         #     {

~/anaconda3/envs/py36/lib/python3.6/site-packages/ccxt/base/exchange.py in extend(*args)
    852                 result = {}
    853             for arg in args:
--> 854                 result.update(arg)
    855             return result
    856         return {}

ValueError: dictionary update sequence element #0 has length 1; 2 is required
```

I can not understand why this error occurs. I trace the source code and everything seems okay.

I extract the problem source code and run it locally, it is okay.

```
import collections
def extend(*args):
    if args is not None:
        result = None
        if type(args[0]) is collections.OrderedDict:
            result = collections.OrderedDict()
        else:
            result = {}
        for arg in args:
            result.update(arg)
        return result
    return {}

request = {
    'currency': "USDT",
}
params = {}
extend(request, params)
```

Especially, I also try FTX for deposit address and I can get correct response. FTX also use *self.extend* and everything is okay...

**Comment by moqiguzhu** [comment]
I found solution for the problem, I believe it's a bug.

```
  def fetch_deposit_address(self, code, params={}):
      rawNetwork = self.safe_string(params, 'network')
      params = self.omit( 'network')
      response = self.fetch_deposit_addresses_by_network(code, params)
      networks = self.safe_value(self.options, 'networks', {})
      network = self.safe_string(networks, rawNetwork, rawNetwork)
```

**params = self.omit( 'network')** should be **params = self.omit( params, 'network')**, if not, params will be **"network"**, and dict.update(a_str) will result in **dictionary update sequence element #0 has length 1; 2 is required**

**Comment by kroitor** [comment]
@moqiguzhu thanks for reporting it! Fixed in 1.61.16+. The new build will arrive in ~20 minutes.
